guides/v2.2/install-gde/install/cli/install-cli-subcommands-lock.md

```diff
@@ -0,0 +1,42 @@
+---
+group: installation-guide
+title: Configure the lock
+functional_areas:
+  - Install
+  - System
+  - Setup
+---
+
+## First steps {#instgde-cli-before}
+{% include install/first-steps-cli.md %}
+In addition to the command arguments discussed here, see [Common arguments]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands.html#instgde-cli-subcommands-common).
+
+## Prerequisites {#instgde-cli-subcommands-store-prereq}
+
+Before you run this command, you must do all of the following *or* you must [install the Magento software]({{ page.baseurl }}/install-gde/install/cli/install-cli-install.html):
+
+*	[Create or update the deployment configuration]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-deployment.html)
+*	[Create the Magento database schema]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-db.html)
+
+{% include install/fully-secure.md %}
+
+## Configure the lock {#instgde-cli-lockconfig}
+
+Since Magento 2.2.9 and 2.3.2 you can configure lock provider.
```

Comment:
```suggestion
Configure a lock provider to prevent the launch of duplicate cron jobs and cron groups. (Requires Magento 2.2.9 and later 2.2.x versions or version 2.3.2 and later.)
```